AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -532,14 +513,16 @@ - (void)testNextBatchSentIfPendingQueueGotRoomAgain {
                 storage:storageMock
           configuration:config
       logsDispatchQueue:dispatch_get_main_queue()];
-  [sut setAppSecret:MS_UUID_STRING];
 
   // When
   [sut enqueueItem:[self getValidMockLog]];
 
   // Try to release one batch.
   dispatch_async(self.logsDispatchQueue, ^{
-    ingestionBlock([@(1) stringValue], 200, nil, nil);
+    XCTAssertNotNil(ingestionBlock);
+    if (ingestionBlock) {
```

Comment:
Redundant. Any warnings for this one?

Reply:
Disagree. This makes assert instead of test crash if something goes wrong. You can see the same usage in the similar places in the file.

Reply:
Oh.. This is async block. Yes, keep it as it is, I missed just one live above.
=> If `ingestionBlock` is `nil`, it should stop the test at that time unless it is not running in the same thread.